packages/main/src/themes/CheckBox.css

```diff
@@ -21,6 +21,7 @@
 
 :host([disabled]) .ui5-checkbox-root {
 	opacity: .5;
```

Comment:
Can we remove `opacity: .5;` now as we use the `--_ui5_checkbox_disabled_opacity` variable?